tests/core/pyspec/eth2spec/test/helpers/hive.py

```diff
@@ -0,0 +1,48 @@
+from ruamel.yaml import YAML
+from dataclasses import dataclass, field
+from typing import Dict
+yaml = YAML()
+
+
+class StateID(str):
+
+    @classmethod
+    def Root(cls, root):
+        return cls(f"root:{root.hex()}")
+
+    @classmethod
+    def Slot(cls, slot):
+        return cls(f"slot:{slot}")
+
+    @classmethod
+    def Head(cls):
+        return cls("head")
+
+    @classmethod
+    def Genesis(cls):
+        return cls("genesis")
+
+    @classmethod
+    def Finalized(cls):
+        return cls("finalized")
+
+    @classmethod
+    def Justified(cls):
+        return cls("justified")
+
+
+@dataclass(kw_only=True)
```

Comment:
`kw_only` is new in Python3.10 but eth2spec's minimum requirement is Python3.9.

Reply:
Fixed! :)

Also removed the `method` key since the YAML library tags the type of each exported class, and I could use this tag in hive to detect the object type, so it worked out quite nicely.